devops/playbooks/aws-ci-startup.yml

```diff
@@ -8,10 +8,18 @@
     - name: Pull in systemd enabled docker image
       docker_image:
         name: "{{ docker_image_sd_build }}"
+      when: "{{ docker_containers != [] }}"
       tags: docker
+
+    - name: Tor docker proxy
+      docker_container:
+        name: tor
+        image: "hsaito/torbox"
+        state: started
       when: "{{ docker_containers != [] }}"
+      tags: docker
 
-    - name: Start build container
+    - name: Start containers
```

Comment:
Will make these task names more explicit, e.g. `Tor docker proxy` -> `Start Tor docker proxy for app-test environment.`